Description: Force the correct Git Id into the image
 We do not build from a git repo and we want the ECL to correctly give
 the commit use. Please change when changing upstream release.
Author: Peter Van Eynde <pvaneynd@debian.org>
Forwarded: not-needed
Last-Update: 2025-10-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- ecl.orig/src/bare.lsp.in
+++ ecl/src/bare.lsp.in
@@ -40,7 +40,7 @@
 ;;; * Create bogus commit version number
 ;;;
 
-(setq si::+commit-id+ "UNKNOWN")
+(setq si::+commit-id+ "294da20f")
 
 ;;;
 ;;; * Setup cross compilation
